packages/fxa-payments-server/server/lib/server.js

```diff
@@ -153,6 +154,17 @@ module.exports = () => {
   );
 
   if (config.get('csp.enabled')) {
+    app.use(function (req, res, next) {
+      // Generate nonce for CSP to allow paypal inline script.
+      res.paypalCspNonce = uuid();
+      next();
```

Comment:
I think we need this on every page, but I'm not certain.  @biancadanforth We need to load the PayPal JS SDK on the subscription management page so we can have a link/button to take the customer to PayPal correct?

Reply:
yeah, right now this is running on every single request. But we can add some paths so it only runs on certain paths.

Reply:
I don't think we would use the PayPal JS SDK for Subscription Management per https://jira.mozilla.com/browse/FXA-2806, which suggests there's an external URL we would send people to for changing their payment method. I haven't looked into the Subscription Management part as closely yet...

Reply:
@bbangert is there any reason we'd need to use the Smart Button for Subscription Management? Maybe there's a scenario I'm forgetting...